pages/initiatives/gsoc/gsoc2022ideas.md

```diff
@@ -1184,6 +1184,8 @@ This will be the initial version of the project. We can advance it more using de
 * Ability for users to report links online with personal information where they can have it removed
 
 Reach out to us on Slack to discuss further on the scope, changes required, or if you have any other proposal.
+* please submit your proposal on the BLT GitHub discussion board.  Because it will be easier for the team to review and give feedback there.
```

Comment:
```suggestion
* Please submit your proposal on the BLT GitHub discussion board.  Because it will be easier for the team to review and give feedback there.
```